src/libraries/System.Net.Http/src/System/Net/Http/MultipartContent.cs

```diff
@@ -281,7 +281,7 @@ private async ValueTask<Stream> CreateContentReadStreamAsyncCore(bool async, Can
                     }
                     else
                     {
-                        readStream = nestedContent.ReadAsStream();
+                        readStream = nestedContent.ReadAsStream(cancellationToken);
```

Comment:
@ManickaP is it ok to pass the token to this method?

Reply:
Yes, it should be OK, thank you!

Reply:
@ManickaP I forgot to include the other instance of `ReadAsStream` not passing a token. It's here:

https://github.com/dotnet/runtime/blob/7d9ec591807254b9a685feb40fcdd6c6a952c19f/src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/DecompressionHandler.cs#L160-L163

Looks quite similar to the other case and it was introduced in the same PR.
Let me know if you don't think I should pass a token in this case.